src/benchmarks/micro/libraries/Microsoft.Extensions.Http/CreationOverheadBenchmark.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.Extensions.DependencyInjection;
+using MicroBenchmarks;
+
+namespace Microsoft.Extensions.Http.Performance
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class CreationOverheadBenchmark
```

Comment:
similar to `LoggingOverheadBenchmark` I believe that this benchmark does not belong here and should be removed